src/plots/cartesian/dragbox.js

```diff
@@ -347,7 +347,7 @@ module.exports = function dragBox(gd, plotinfo, x, y, w, h, ns, ew) {
     var scrollViewBox = [0, 0, pw, ph],
         // wait a little after scrolling before redrawing
         redrawTimer = null,
-        REDRAWDELAY = 300,
+        REDRAWDELAY = 50,
```

Comment:
:+1: for making this smaller.

But, could you move this `REDRAWDELAY` value to `plots/cartesian/constants` so that power users can easily change it if they want slower redraw cycles.